### DISCUSSION TITLE
Network config path configs\nerf\base.json does not exist.

### DISCUSSION THREAD

**Discussion by jessekirbs** [NVlabs/instant-ngp#395] [Q&A] [answered]
For some reason, now when I try to run `testbed.exe --scene /path/` I get the following error:

```
19:52:30 INFO     Loading NeRF dataset from
19:52:30 INFO       c:\tools\volumetric\instant-ngp\data\bedrock\transforms.json
19:52:30 SUCCESS  Loaded 70 images of size 1440x1080 after 0s
19:52:30 INFO       cam_aabb=[min=[-1.05447,-0.994622,0.410848], max=[2.11579,2.57024,1.48582]]
19:52:30 ERROR    Network config path configs\nerf\base.json does not exist.
```
As far as I know, nothing was changed. I was getting memory errors before, then I tried running the 'fox' example and got this error. Now I get it on any datasets. Any ideas?

**Comment by Tom94** [comment, marked as answer]
Hi, you're probably not launching the program from the projects root directory, which means it can't find the default relative path `configs\nerf\base.json`. You can pass a network config path manually using the `--network` options (`--help` for more info about CLI arguments).

**Reply by jessekirbs** [reply]
Ah, of course. Thank you, @Tom94 !

**Reply by jiajigiser** [reply]
Run the command line from the previous directory and enter the code again
